runtime/oti/vm_api.h

```diff
@@ -2925,8 +2925,8 @@ getStringUTF8Length(J9VMThread *vmThread,j9object_t string);
 /**
 * Check incoming class name characters and return following values accordingly:
 * 	CLASSNAME_INVALID - if there is a character '/';
-* 	CLASSNAME_VALID_NON_ARRARY - if it is valid and there is no '[' at beginning of class name string;
-* 	CLASSNAME_VALID_ARRARY - if it is valid and there is a '[' at beginning of class name string.
+* 	CLASSNAME_VALID_NON_ARRAY - if it is valid and there is no '[' at beginning of class name string;
+* 	CLASSNAME_VALID_ARRAY - if it is valid and there is a '[' at beginning of class name string.
```

Comment:
These reflect (mis-spelled) constants and should wait until the constants are changed.